core-modular/src/services/i18n.service.ts

```diff
@@ -138,13 +138,12 @@ export class i18nService {
    * Sets locale to all Luigi containers
    */
   private broadcastLocaleToAllContainers(locale: string): void {
-    const containerWrapper = this.luigi.getEngine()._connector?.getContainerWrapper();
+    const containers = document.querySelectorAll('luigi-container');
```

Comment:
Discuss in daily
const containers = Array.from(document.querySelectorAll('[class^="luigi-"]'))
  .filter(el => Array.from(el.classList)
    .some(c => c.endsWith('-container'))
  );